### PR TITLE
decouple PerspectiveCamera parameters in ObjectLoader

### PR DESCRIPTION
**Pull request by bhouston**
Just improving long-term robustness in the read by removing unnecessary coupling between PerspectiveCamera parameters.

For more discussion see here: https://github.com/mrdoob/three.js/pull/8561/files/7eea5d47d86a6376175761cddab2742f17de177f#r59205742

### CODE REVIEW

File: src/loaders/ObjectLoader.js [hunk 1]
```diff
@@ -471,16 +471,9 @@ THREE.ObjectLoader.prototype = {
...
-						if ( data.view !== null )
-							object.view = Object.assign( {}, data.view );
-
-						object.filmGauge = data.filmGauge;
-						object.filmOffset = data.filmOffset;
-
-					}
+					if ( data.filmGauge !== undefined ) object.filmGauge = data.filmGauge;
+					if ( data.filmOffset !== undefined ) object.filmOffset = data.filmOffset;
+					if ( data.view !== undefined ) object.view = Object.assign( {}, data.view );
```
Comment (tschw) [inline src/loaders/ObjectLoader.js]: Breaking, since `Object.assign( {}, null )` yields an empty object, not `null`.

Comment (tschw) [inline src/loaders/ObjectLoader.js]: `null` is the default, so testing for `data.view !== undefined && data.view !== null` or just `data.view != null` will fix it.

**Comment by tschw** [comment]
See #8561 (for cross-reference).

File: src/loaders/ObjectLoader.js (hunk 1 above)
Comment (mrdoob) [inline src/loaders/ObjectLoader.js]: We should probably not serialise `view` when is `null`.

Comment (bhouston) [inline src/loaders/ObjectLoader.js]: > We should probably not serialise view when is null.

Agreed.  I just updated the PR to do that.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**